DecipherNow/gm-fabric-dashboard
# refactor: cleanup baseURL props now that fabricServer is guaranteed

Resolves #1321 

## Proposed Changes

  - Remove logic that conditionally used baseURL if running with fabricServer to always use baseURL
  - Perform minor cleanup and refactoring
  
## How to Test

  - Carefully review changes
  - npm test
  - Run app to confirm things work as before
